mimic/core.py

```diff
@@ -164,6 +192,7 @@ def entries_for_tenant(self, tenant_id, prefix_map, base_uri):
                     prefix_map[endpoint] = api.uri_for_service(
                         endpoint.region, service_id
                     )
+
```

Comment:
It doesn't seem like this newline is necessary; if not, would you mind removing? :-)